spec/features/users/logout_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.describe 'User can log out' do
+  describe "as a default user" do
+     it 'can log out as a registered user' do
+       default_user = User.create!(name: "Johnny",
+                                   street_address: "123 Jonny Way",
+                                   city: "Johnsonville",
+                                   state: 'TN',
+                                   zip_code: 12345,
+                                   email: "roman@example.com",
+                                   password: "hamburger01",
+                                   role: 0
+                                 )
+
+        allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(default_user)
+
+        @meg = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+
+        @tire = @meg.items.create(name: "Gatorskins", description: "They'll never pop!", price: 100, image: "https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 12)
+
+        visit "/items/#{@tire.id}"
+        click_on "Add To Cart"
+        click_on "Log Out"
```

Comment:
May want to add in an additional test for the cart incrementation before logging out as well to ensure there are items in the cart before log out